ui/create_restore_page.go

```diff
@@ -382,30 +382,33 @@ func (pg *createRestore) editorSeedsEventsHandler() {
 			case widget.ChangeEvent:
 				pg.seedSuggestionButtonWidgets = nil
 				pg.seedSuggestionButtons = nil
+				pg.initSeedSuggestionButtonWidgets(editor)
 
-				if strings.Trim(editor.Text(), " ") == "" {
-					return
+			case widget.SubmitEvent:
+				if i < len(pg.seedEditorWidgets.editors)-1 {
+					pg.seedEditorWidgets.editors[i+1].Focus()
 				}
+			}
+		}
+	}
+}
 
-				for _, word := range dcrlibwallet.PGPWordList() {
-					if strings.HasPrefix(strings.ToLower(word), strings.ToLower(editor.Text())) {
-						if len(pg.seedSuggestionButtonWidgets) < 2 {
-							var btn struct {
-								text   string
-								button widget.Button
-							}
+func (pg *createRestore) initSeedSuggestionButtonWidgets(editor *widget.Editor) {
```

Comment:
no need to add "widgets" to the method name. "initSeeSuggestionButtons" is quite fine.